src/cryptography/hazmat/backends/openssl/x509.py

```diff
@@ -58,6 +59,14 @@ def version(self):
 
     @property
     def serial(self):
+        warnings.warn(
+            "Certificate serial is deprecated, use serial_number instead.",
+            DeprecationWarning
```

Comment:
This should be a constant in the `cryptography.utils` module you'll see a few other examples there.